src/api_utils/gladia_api_utils/submodules.py

```diff
@@ -195,7 +195,15 @@ def exec_in_subprocess(
             executable="/bin/bash",
         )
 
-        proc.communicate()
+        _, error_message = proc.communicate()
+
+        # if the subprocess has failed (return code of shell != 0)
+        # raise an exception and log to the console the error message
+        if proc.returncode != 0:
+            logger.error(error_message)
+            raise RuntimeError(
+                f"Could not run in subprocess command {cmd}: {error_message}"
```

Comment:
```suggestion
                f"Subprocess encountered the following error : {error_message}\nCommand executed: {cmd}"
```

I find it more explicit but up to you, doesn't really matter that much